src/ima.js

```diff
@@ -571,24 +570,16 @@ class Ima extends BasePlugin implements IMiddlewareProvider, IAdsControllerProvi
   }
 
   /**
-   * Initializing the VpaidMode.
+   * Gets the vpaid mode.
    * @private
-   * @returns {string} - The VPAIDMode type
+   * @returns {string} - The vpaid mode.
    * @instance
    * @memberof Ima
    */
   _getVpaidMode(): string {
-    switch (this.config.vpaidMode) {
-      case VpaidMode.ENABLED:
-        return this._sdk.ImaSdkSettings.VpaidMode.ENABLED;
-      case VpaidMode.INSECURE:
-        return this._sdk.ImaSdkSettings.VpaidMode.INSECURE;
-      case VpaidMode.DISABLED:
-        return this._sdk.ImaSdkSettings.VpaidMode.DISABLED;
-      default:
-        return this._sdk.ImaSdkSettings.VpaidMode.ENABLED;
-    }
+    return this.config.vpaidMode || this._sdk.ImaSdkSettings.VpaidMode.ENABLED;
```

Comment:
Do you think that should be OK instead?

`_getVpaidMode(): string {
    const vpaidmode = window.google.ima.ImaSdkSettings.VpaidMode[this.config.vpaidMode];
    if (this.config.vpaidMode && typeof vpaidmode === 'number') {
      this.logger.debug('VpaidMode: set to ' + this.config.vpaidMode);
      return vpaidmode;
    } else {
      this.logger.warn('VpaidMode is not set, setting to ENABLED');
      return this._sdk.ImaSdkSettings.VpaidMode.ENABLED;
    }
  }`